nvse/nvse/CachedScripts.cpp

```diff
@@ -78,18 +78,24 @@ void CacheAllScriptsInPath(std::string_view pathStr)
 	{
 		for (auto const& dir_entry : std::filesystem::recursive_directory_iterator(path))
 		{
-			if (dir_entry.is_regular_file())
+			if (dir_entry.is_regular_file() && (dir_entry.path().has_extension())) //will make it so that it will not compile script file that doesn't have file extension
 			{
 				if (!CompileAndCacheScript(dir_entry.path(), false))
 				{
-					std::string errMsg = std::format("xNVSE: Failed to precompile script file {}", 
+					std::string errMsg = std::format("xNVSE: Failed to precompile script file at {}",
 						dir_entry.path().string());
 					Console_Print(errMsg.c_str());
 					_ERROR(errMsg.c_str());
 				}
+				else
+				{
+					std::string compileSuccessMsg = std::format("xNVSE: script file successfully precompiled at {}",
+						dir_entry.path().string());
+					Console_Print(compileSuccessMsg.c_str());
+					_ERROR(compileSuccessMsg.c_str());
```

Comment:
Should be _MESSAGE, not _ERROR.
Also personally I'd remove the Console_Print, otherwise it will completely clog the console for certain setups.

Reply:
ok